moler/cmd/at/get_functionality_level.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+AT+CFUN
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Kamil Pielka'
+__copyright__ = 'Copyright (C) 2020, Nokia'
+__email__ = 'kamik.pielka@nokia.com'
+
+import re
+
+from moler.cmd.at.genericat import GenericAtCommand
+from moler.exceptions import ParsingDone
+
+
+class GetFunctionalityLevel(GenericAtCommand):
+    """
+    Command to get UE's level of functionality. Example output:
+
+    +CFUN: 1
+
+    OK
+    """
+    def __init__(self, connection=None, prompt=None, newline_chars=None, runner=None):
+        """Create instance of GetFunctionalityLevel class"""
+        super(GetFunctionalityLevel, self).__init__(connection, operation='execute', prompt=prompt,
+                                                    newline_chars=newline_chars, runner=runner)
+
+    def build_command_string(self):
+        return "AT+CFUN?"
+
+    def on_new_line(self, line, is_full_line):
+        """
+        Method to parse command output. Will be called after line with command echo.
+
+        +CFUN: 1
+
+        OK
+
+        Write your own implementation but don't forget to call on_new_line from base class
+
+        :param line: Line to parse, new lines are trimmed
+        :param is_full_line:  False for chunk of line; True on full line (NOTE: new line character removed)
+        :return: None
+        """
+        if is_full_line:
+            try:
+                self._parse_fun_level(line)
+            except ParsingDone:
+                pass
+        return super(GetFunctionalityLevel, self).on_new_line(line, is_full_line)
+
+    _re_fun_level = re.compile(r'^\s*\+CFUN:\s*(?P<fun_level>[\d+]{1,3})\s*$')
```

Comment:
Please add comment with example of any string to match by this regex.

Reply:
Example of string added